lib/prompts/input.js

```diff
@@ -82,7 +83,7 @@ Prompt.prototype.onEnd = function( state ) {
     this.status = "answered";
 
     // Re-render prompt
-    this.clean(1).render();
+    this.clean( this.lines() ).render();
```

Comment:
@SBoudrias If you take a look at my latest changes in the PR, I was able to unit test `Prompt.prototype.onKeypress()` by using `this.rl.emit()`, however calling `this.rl.emit()` in the unit tests doesn't call `Prompt.prototype.onEnd()` (it does call it when not mocking readline), so I wonder if you know a way to make `Prompt.prototype.onEnd()` get called as well.